proposals/csharp-12.0/collection-expressions.md

```diff
@@ -167,9 +170,14 @@ For the *create method*:
 * The method must be `static`.
```

Comment:
This bullet and the previous one can be pulled out of the fork, that I am suggesting to add, and shared